engine/src/main/java/com/datatorrent/stram/security/StramWSFilterInitializer.java

```diff
@@ -59,9 +59,11 @@ public void initFilter(FilterContainer container, Configuration conf)
       for (String rmId : ConfigUtils.getRMHAIds(conf)) {
         proxies.add(getResolvedRMWebAppURLWithoutScheme(conf, rmId));
       }
+      logger.info("HA proxy addresses {}", proxies);
```

Comment:
It's a one time message. Thought it would be easier to find the problem if the information is available right away.